Library/Homebrew/cmd/cleanup.rb

```diff
@@ -108,10 +108,7 @@ def cleanup_lockfiles
   end
 
   def rm_DS_Store
-    paths = %w[Cellar Frameworks Library bin etc include lib opt sbin share var].
```

Comment:
@geoff-codes If you have code like e.g. `an_array.map(something).flatten`, the only way to split it on multiple lines is like this:

```ruby
an_array.
map(something).
flatten
```